docs/source/metadata.rst

```diff
@@ -8,29 +12,67 @@ or an e-mail with a given subject line.
+| ``file_directory``          | File Directory                                           |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``last_modified``           | Last Modified Date                                       |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``filetype``                | File Type                                                |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``type``                    | Element Type                                             | Categorizes elements into types such as Title, NarrativeText. Not a metadata field                                                                                                                                                                                                          |
```

Comment:
need to be careful here as the `type` technically lives outside of `metadata` even though downstream, for all intents and purposes, it kind of is.

https://github.com/Unstructured-IO/unstructured/blob/19d8bff/test_unstructured_ingest/expected-structured-output/s3/small-pdf-set/2023-Jan-economic-outlook.pdf.json#L3

Maybe change the first column to `N/A` and update Details to mention the caveat?

Reply:
+1 on `type` not actually being in metadata. I removed it from the original table since I noticed the very top of the notion doc was "Types (Categories)" - I think it's best to mention its availability separately (like [here](https://unstructured-io.github.io/unstructured/introduction.html#document-elements) in the docs and remove it from this table.